Doc/library/datetime.rst

```diff
@@ -1293,48 +1458,67 @@ Instance methods:
 
 .. method:: datetime.ctime()
 
-   Return a string representing the date and time, for example ``datetime(2002, 12,
-   4, 20, 30, 40).ctime() == 'Wed Dec  4 20:30:40 2002'``. ``d.ctime()`` is
-   equivalent to ``time.ctime(time.mktime(d.timetuple()))`` on platforms where the
-   native C :c:func:`ctime` function (which :func:`time.ctime` invokes, but which
-   :meth:`datetime.ctime` does not invoke) conforms to the C standard.
+   Return a string representing the date and time::
+
+       >>> from datetime import datetime
+       >>> datetime(2002, 12, 4, 20, 30, 40).ctime()
+       'Wed Dec  4 20:30:40 2002'
 
+   The output string will *not* include time zone information, regardless
+   of whether the input is aware or naive.
+
+   ``d.ctime()`` is equivalent to::
+
+     time.ctime(time.mktime(d.timetuple()))
+
+   on platforms where the native C :c:func:`ctime` function
+   (which :func:`time.ctime` invokes, but which
+   :meth:`datetime.ctime` does not invoke) conforms to the C standard.
 
 .. method:: datetime.strftime(format)
 
    Return a string representing the date and time, controlled by an explicit format
-   string.  For a complete list of formatting directives, see
+   string. For a complete list of formatting directives, see
    :ref:`strftime-strptime-behavior`.
 
 
 .. method:: datetime.__format__(format)
 
-   Same as :meth:`.datetime.strftime`.  This makes it possible to specify a format
+   Same as :meth:`.datetime.strftime`. This makes it possible to specify a format
    string for a :class:`.datetime` object in :ref:`formatted string
-   literals <f-strings>` and when using :meth:`str.format`.  For a
+   literals <f-strings>` and when using :meth:`str.format`. For a
    complete list of formatting directives, see
    :ref:`strftime-strptime-behavior`.
 
+:class:`datetime`: Examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Examples of working with datetime objects:
+Examples of working with :class:`~datetime.datetime` objects:
 
 .. doctest::
 
     >>> from datetime import datetime, date, time
+
     >>> # Using datetime.combine()
     >>> d = date(2005, 7, 14)
     >>> t = time(12, 30)
-    >>> datetime.combine(d, t)
+    >>> combined = datetime.combine(d, t)
+    >>> combined
     datetime.datetime(2005, 7, 14, 12, 30)
+    >>> combined.month, combined.day, combined.hour
```

Comment:
I'm not sure I understand what you are trying to demonstrate here. I think if you want to show that you can access arbitrary components of a given datetime, it might be better to not combine that with the `datetime.combine` example.